src/Components/Web/test/WebEventData/KeyboardEventArgsReaderTest.cs

```diff
@@ -22,6 +22,7 @@ public void Read_Works()
             Repeat = true,
             ShiftKey = true,
             Type = "type1",
+            IsComposing = false,
```

Comment:
```suggestion
            IsComposing = true,
```

It's more useful to test the `true` case, because the default value is `false`, so if we test `false`, it would pass even if we removed the parsing implementation.